src/globals.js

```diff
@@ -2,18 +2,26 @@ import {
   Config,
   Translation,
   withTranslation as withTranslationImpl,
-} from 'wundery-js-lib';
-import * as translations from 'translations';
+} from "wundery-js-lib";
+import * as translations from "translations";
 
 // Build translation instance
 export const translation = new Translation({
   dictionaries: translations,
   locale: Translation.guessLocaleFromBrowser(),
-  fallbackLocale: 'en',
+  fallbackLocale: "en",
 });
 
 // Build a HOC wrapping the withTranslation HOC by injecting the translation instance
-export const withTranslation = (...args) => withTranslationImpl(...[translation].concat(args));
+export const withTranslation = (...args) =>
+  withTranslationImpl(...[translation].concat(args));
 
 // Build a central config object
 export const config = new Config({ version: VERSION });
+
+export const loadInfinityClasses = {
+  category: ".category",
+  loadMore: ".load-more",
+  footer: ".footer-wrapper",
+  appendProducts: ".products",
+};
```

Comment:
Just hardcoding these is not how this is suppsed to work: these are provided by the theme as a configuration option, currently in the wundery theme via the `templates/global_variables.js`. With this implementation, the user can't change these selectors anymore so this would fail if he uses different classes in his theme.

Please take a look at how the `newSearch` and related functions of frontline are implemented: they accept an object with configuration values and return an object which can be `mount()`ed into the dom.